card/mod/01_core/set/all/permissions.rb

```diff
@@ -72,7 +72,7 @@ def deny_because why
 
 def permitted? action
 
-  if !Wagn.config.read_only
+  if !Cardio.config.read_only
```

Comment:
I still think, API-wise, it would be nice if we could access this as `Card.config.read_only`.  I know there are some loading contexts where this won't work and Cardio is needed.  But can't we just add this to the card class somewhere:

````
def config
  Cardio.config
end
````
...and use that in mod contexts?